install

```diff
@@ -157,12 +161,17 @@ chmods = %w( . bin etc include lib lib/pkgconfig Library sbin share var var/log
              share/man/man5 share/man/man6 share/man/man7 share/man/man8
              share/info share/doc share/aclocal ).
              map { |d| File.join(HOMEBREW_PREFIX, d) }.select { |d| chmod?(d) }
+chowns = chmods.select { |d| chown?(d) }
 chgrps = chmods.select { |d| chgrp?(d) }
 
 unless chmods.empty?
   ohai "The following directories will be made group writable:"
   puts(*chmods)
 end
+unless chowns.empty?
+  ohai "The following directories will have their owner set to #{Tty.underline 39}#{ENV['USER']}#{Tty.reset}:"
```

Comment:
We should put this `Tty.underline 39` in a variable with a meaningful name; it’s hard to guess what it means here.